### PR TITLE
Add output flag for spire-server mint and generate commands

### PR DESCRIPTION
**Pull request by guilhermocc**
**Pull Request check list**

- [x] Commit conforms to CONTRIBUTING.md?
- [x] Proper tests/regressions included?
- [x] Documentation updated?

**Affected functionality**
spire-server `mint` and `genetate` commands,

**Description of change**
add `-output` flag in spire-server `mint` and `generate` commands.

**Which issue this PR fixes**
Ongoing work for #1354

### CODE REVIEW

File: cmd/spire-server/cli/x509/mint.go [hunk 1]
```diff
@@ -182,3 +170,55 @@ func (c *mintCommand) Run(ctx context.Context, env *common_cli.Env, serverClient
 func ttlToSeconds(ttl time.Duration) int32 {
 	return int32((ttl + time.Second - 1) / time.Second)
 }
+
+type MintResult struct {
```
Comment (amartinezfayo) [inline cmd/spire-server/cli/x509/mint.go:174]: It doesn't seem that we need to export this, should be unexported.

**Review by amartinezfayo — COMMENTED** [review]
Thank you @guilhermocc for this, it's looking great!

File: cmd/spire-server/cli/x509/mint.go (hunk 1 above)
Comment (guilhermocc) [reply cmd/spire-server/cli/x509/mint.go:174]: You're right. Thank you for the review @amartinezfayo 😄

Comment (guilhermocc) [reply cmd/spire-server/cli/x509/mint.go]: 238ceef

**Review by amartinezfayo — APPROVED** [review]
Thank you @guilhermocc!

**Comment by guilhermocc** [comment]
Thank you for the Review, @amartinezfayo

**Pull request merged**